### PR TITLE
Correct instructions to help readers find template

### PR DESCRIPTION
**Pull request by ilovepullrequests**
The current instructions are out of date. Bouncy Blast is nowhere to be found on the linked page; it's actually under 'Engage Your Audiences'.

### CODE REVIEW

File: src/asciidoc/tutorials/getting-started/your-first-remix.adoc [hunk 1]
```diff
@@ -53,7 +53,7 @@ You’ll see that they are sorted into categories, depending on what they do.
 There are templates focused on social media engagement, memes, and games.
 New templates are being added all the time by developers like you.
 
-Scroll down until you find the Bouncy Blast template.
+Scroll down until you see 'Engage Your Audiences', click on 'See All', then scroll down until you find the Bouncy Blast template.
```
Comment (rasienko) [inline src/asciidoc/tutorials/getting-started/your-first-remix.adoc:56]:
```suggestion
Scroll down until you see *Engage Your Audiences*, click *See All*, then scroll down until you find the Bouncy Blast template.
```

**Review by rasienko — CHANGES_REQUESTED** [review]
Thanks so much @ilovepullrequests ! Sorry I didn't see this PR sooner to finish the review. The content changes are excellent; thanks for finding this outdate info! Just minor updates to the styling to match the rest of our site. You should be able to just accept the suggestion and then I can merge your update to the live site!

**Comment by ilovepullrequests** [comment]
> Thanks so much @ilovepullrequests ! Sorry I didn't see this PR sooner to finish the review. The content changes are excellent; thanks for finding this outdate info! Just minor updates to the styling to match the rest of our site. You should be able to just accept the suggestion and then I can merge your update to the live site!

Done!

**Review by rasienko — APPROVED** [review]
Thanks!

**Comment by rasienko** [comment]
@ilovepullrequests your changes are live! 
https://developer.withkoji.com/tutorials/getting-started/your-first-remix#_creating_a_remix_from_a_template

**Comment by ilovepullrequests** [comment]
> @ilovepullrequests your changes are live!
> https://developer.withkoji.com/tutorials/getting-started/your-first-remix#_creating_a_remix_from_a_template

Amazing!

**Pull request merged**